### PR TITLE
Update ReviewDateWidget.php

### PR DESCRIPTION
**Pull request by rgcarr**
Added default value to language code in formElement()

### CODE REVIEW

File: modules/localgov_review_date/src/Plugin/Field/FieldWidget/ReviewDateWidget.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
 
     // Get current review status object.
     $entity = $items->getEntity();
-    $langcode = $form_state->get('langcode');
+    $langcode = $form_state->get('langcode') ?? 'und';
```
Comment (andybroomfield) [inline modules/localgov_review_date/src/Plugin/Field/FieldWidget/ReviewDateWidget.php]: Can we change `und` to `LanguageInterface::LANGCODE_NOT_SPECIFIED`
You'll also need to add a `use Drupal\Core\Language\LanguageInterface;` statement at the top.

**Comment by andybroomfield** [comment]
Thanks for looking at this @rgcarr.
Please see inline comment, we shouldn't reference und directly as it (in theory) can change in core, so best to use a class constant.

**Comment by rgcarr** [comment]
Thanks for the help @andybroomfield . Changes made and tested locally

**Review by andybroomfield — APPROVED** [review]
Happy for this to be merged, thanks @rgcarr.
I tested this with the views bullk edit module to replicate the error, and the fix provided address that.
I also checked that with the fix that it is still possible to edit nodes with the review date widget, and was able to do so.

**Pull request merged**